echopype/tests/echodata/test_echodata.py

```diff
@@ -267,24 +267,25 @@ def test_getitem(self, converted_zarr):
         ed = EchoData.from_file(converted_raw_path=converted_zarr)
         beam = ed['Sonar/Beam_group1']
         assert isinstance(beam, xr.Dataset)
-        try:
-            ed['MyGroup']
-        except Exception as e:
-            assert isinstance(e, GroupNotFoundError)
+        assert ed['MyGroup'] is None
 
         ed._tree = None
         try:
             ed['Sonar']
         except Exception as e:
             assert isinstance(e, ValueError)
 
-
     def test_setitem(self, converted_zarr):
         ed = EchoData.from_file(converted_raw_path=converted_zarr)
         ed['Sonar/Beam_group1'] = ed['Sonar/Beam_group1'].rename({'beam': 'beam_newname'})
 
         assert sorted(ed['Sonar/Beam_group1'].dims.keys()) == ['beam_newname', 'channel', 'ping_time', 'range_sample']
 
+        try:
+            ed['SomeRandomGroup'] = 'Testing value'
+        except Exception as e:
+            assert isinstance(e, GroupNotFoundError)
+
```

Comment:
I like this!

Reply:
@b-reyes : the `test_setitem` and `test_getitem` are the type of unit tests I was thinking for the changes you made in #774. Just making a note here as it came to my mind while reviewing this. We can circle back to that and your ideas in our tests-hackday!